src/ui/server/helpers/mappings/map-application-to-form-fields/index.ts

```diff
@@ -83,6 +85,13 @@ const mapApplicationToFormFields = (application?: Application): Application | ob
       };
     }
 
+    if (application?.nominatedLossPayee?.[IS_LOCATED_INTERNATIONALLY] || application?.nominatedLossPayee?.[IS_LOCATED_IN_UK]) {
```

Comment:
Could this be simpler with `if(objectHasValues(application?.nominatedLossPayee)) { ...`  ?